src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,153 @@
+CockroachDB is made available under the [CockroachDB Software License][csl].
+
+## Types of licenses
+
+The table of licenses below refers to options for {{ site.data.products.core }} deployments. All {{ site.data.products.cloud }} deployments automatically have a valid **Enterprise** license.
+
+Type | Description
+-------------|------------
+**Enterprise** <a name="enterprise"></a> | This paid license allows usage of all CockroachDB features with no restrictions under the terms specified in the [CockroachDB Software License][csl]. License must be renewed annually or as negotiated. Support levels available include [Enterprise][support] or [Essential][support].
+**Enterprise Free** <a name="enterprise-free"></a> | Same functionality as **Enterprise**, but free of charge for businesses with less than $10M in annual revenue, and telemetry is required except for ephemeral clusters (7 days or less). Clusters will be [throttled](#throttling) after 7 days without sending telemetry. License must be renewed annually. Support level available is Community (i.e., [Docs]({% link {{ page.version.version }}/index.md %}), [Forum][forum], [Slack][slack]).
```

Comment:
If the cluster is ephemeral, you wouldn't need to install a license at all. It might be helpful to explain the "no license" scenario, as users may want to know they have a 7-day grace period before needing to install a license and you don't have to send telemetry in the grace period.

Reply:
I updated the text at the bottom of this section to cover the no license and single-node dev cluster scenarios as follows. PTAL

> Note that:
> - Clusters with no license have a 7-day grace period before needing to install a license and start sending telemetry data.
> - No license key is required for developers running [single-node clusters](#single-node-clusters).